pyproject.toml

```diff
@@ -32,38 +61,172 @@ ignore = """
 tests
 """
 
-[tool.pylint.messages_control]
-disable = """
-bad-continuation,line-too-long,too-few-public-methods,import-error,too-many-instance-attributes, too-many-arguments,
-raise-missing-from, invalid-name, too-many-lines
-"""
+[build-system]
+requires = ["poetry>=0.12"]
+build-backend = "poetry.masonry.api"
+
+[tool.isort]
+multi_line_output = 3
+include_trailing_comma = true
+force_grid_wrap = 0
+use_parentheses = true
+line_length = 120
+
+[tool.black]
+target-version = ["py310"]
+line-length = 120
+include = '\.pyi?$'
+exclude = '''
+(
+  /(
+      \.eggs         # exclude a few common directories in the
+    | \.git          # root of the project
+    | \.hg
+    | \.mypy_cache
+    | \.tox
+    | \.venv
+    | _build
+    | buck-out
+    | build
+    | dist
+  )/
+  | foo.py           # also separately exclude a file named foo.py in
+                     # the root of the project
+)
+'''
+
+[tool.pytest.ini_options]
+addopts = [
+    "--import-mode=importlib",
+]
+log_cli = true
+log_cli_level = "INFO"
 
 [tool.pylint.basic]
 good-names = """
 logger,i,j,k,ex,Run,_
 """
 
-[tool.poetry.dependencies]
-python = "^3.6.2"
-requests = "^2.21"
-email_validator = "^1.0"
-pydantic = "^1.4"
-typing_extensions = [
-    { version = "^3.7.4", python = "~3.6" },
-    { version = "^4", python = "^3.7" },
+[tool.pylint.messages_control]
```

Comment:
No need since pylint is not used any more

Reply:
here

Reply:
done